datalake-dr-connector/src/main/java/com/sequenceiq/cloudbreak/datalakedr/config/DatalakeDrConfig.java

```diff
@@ -0,0 +1,52 @@
+package com.sequenceiq.cloudbreak.datalakedr.config;
+
+import javax.annotation.PostConstruct;
+
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class DatalakeDrConfig {
+
+    private static final int DEFAULT_DATALAKE_DR_PORT = 80;
+
+    @Value("${altus.datalakedr.endpoint:}")
+    private String endpoint;
+
+    private String host;
+
+    private int port;
+
+    @PostConstruct
+    public void init() {
+        if (isConfigured()) {
+            String[] parts = endpoint.split(":");
+            if (parts.length < 1 || parts.length > 2) {
+                throw new IllegalArgumentException("altus.datalakedr.endpoint must be in host or host:port format.");
+            }
+            host = parts[0];
+            port = parts.length == 2
+                ? Integer.parseInt(parts[1])
+                : DEFAULT_DATALAKE_DR_PORT;
+        } else {
+            throw new IllegalStateException("altus.datalakedr.endpoint is not configured");
```

Comment:
Does this mean that the application won't start if we're not configuring this endpoint? If that's the case we will need to introduce the datalake dr application in our dev Kubernetes cluster as well along with the local CBD. I would rather create another endpoint to enable/disable this feature so it's not a hard dependency. If it's just not configured then we're skipping some of steps that you do as part of this PR. We do similar things with Cluster Proxy, Public Endpoint Management service and with other external services as well.

Reply:
I tried running the CB application and it fails with the endpoint not being set. Probably we should create a datalakedr.enabled variable also with a `false` default value and safeguard all calls to only make them if it's enabled.

Reply:
I've removed the datalake-dr-connector dependency from core/build.gradle. I've also removed the colon from `${altus.datalakedr.endpoint}`, included the default value of an empty string in the datalake application.yml file. Finally, I added a new setting `altus.datalakedr.enabled` that defaults to false if it's not set, and will prevent `DatalakeDrConfig` from throwing an exception on `init()` if it's not enabled. Please let me know if that's working for you.